protocols/v1/src/error.rs

```diff
@@ -6,6 +6,7 @@ pub enum Error {
     Method(MethodError),
     InvalidJsonRpcMessageKind,
     InvalidSubmission,
+    InvalidState(String),
```

Comment:
I added this for debugging - LMK if you don't want to keep it or if you had something else in mind.